gfauto/gfauto/fuzz_glsl_test.py

```diff
@@ -179,18 +195,34 @@ def make_test(
     subtest_dir: Path,
     spirv_opt_args: Optional[List[str]],
     binary_manager: binaries_util.BinaryManager,
+    derived_from: Optional[str],
+    stable_shader: bool,
 ) -> Path:
+
+    source_dir = test_util.get_source_dir(subtest_dir)
+
     # Create the subtest by copying the base source.
-    util.copy_dir(base_source_dir, test_util.get_source_dir(subtest_dir))
+    util.copy_dir(base_source_dir, source_dir)
 
-    test = Test(glsl=TestGlsl(spirv_opt_args=spirv_opt_args))
+    test = Test(glsl=TestGlsl(spirv_opt_args=spirv_opt_args), derived_from=derived_from)
 
     test.binaries.extend([binary_manager.get_binary_by_name(name="glslangValidator")])
     add_spirv_shader_test_binaries(test, spirv_opt_args, binary_manager)
 
     # Write the test metadata.
     test_util.metadata_write(test, subtest_dir)
 
+    # If the reference shader is "stable" (with respect to floating-point sensitivity)
+    # then this can be a wrong image test. I.e. we will render the reference and variant shaders
```

Comment:
Nit: I would not start a sentence "I.e." - I would change "test. I.e." to "test; i.e." here.